pkg/services/accesscontrol/accesscontrol.go

```diff
@@ -34,6 +34,8 @@ type Service interface {
 	DeclareFixedRoles(registrations ...RoleRegistration) error
 	//IsDisabled returns if access control is enabled or not
 	IsDisabled() bool
+	// DeleteUserAccessControl deletes access control for user
+	DeleteUserAccessControl(context.Context, int64) error
```

Comment:
What is the difference between this new function and DeleteUserPermissions?

Reply:
I moved `DeleteUserAccessControl` from sqlstore to AC - as I am deleting sqlstore, but if there is another method with the same functionality I am happy to delete it @kalleep

Reply:
Yeah so I think we can just use DeleteUserPermissions and not add another function that I think is doing the same thing